apps/dashboard/src/app/team/[team_slug]/[project_slug]/connect/analytics/_components/DailyConnectionsChartCard.tsx

```diff
@@ -0,0 +1,148 @@
+import {
+  type ChartConfig,
+  ChartContainer,
+  ChartTooltip,
+  ChartTooltipContent,
+} from "@/components/ui/chart";
+import {
+  Select,
+  SelectContent,
+  SelectItem,
+  SelectTrigger,
+  SelectValue,
+} from "@/components/ui/select";
+import type { WalletStats } from "@3rdweb-sdk/react/hooks/useApi";
+import { format } from "date-fns";
+import { useMemo, useState } from "react";
+import { Bar, BarChart, CartesianGrid, LabelList, XAxis } from "recharts";
+import { EmptyChartState, LoadingChartState } from "./EmptyChartState";
+
+type ChartToShow = "uniqueWallets" | "totalWallets";
```

Comment:
could this just be `typeof chartToShowoptions[number]` ?

Reply:
Not sure what you mean, `ChartToShow` is the source type and everything else is defined using it